storage-s3-dynamodb/src/main/java/io/delta/storage/S3DynamoDBLogStore.java

```diff
@@ -113,14 +115,31 @@ public S3DynamoDBLogStore(Configuration hadoopConf) throws IOException {
             "com.amazonaws.auth.DefaultAWSCredentialsProviderChain"
         );
         regionName = getParam(hadoopConf, DDB_CLIENT_REGION, "us-east-1");
+
+        final String ttl = getParam(hadoopConf, TTL_SECONDS, null);
+        expirationDelaySeconds = ttl == null ?
+            BaseExternalLogStore.DEFAULT_EXTERNAL_ENTRY_EXPIRATION_DELAY_SECONDS :
+            Long.parseLong(ttl);
+        if (expirationDelaySeconds < 0) {
```

Comment:
Worth to define a minimal expiration time larger thank 0 and validate it here.

Reply:
What should that minimal expiration time be?

Reply:
We don't plan to document this flag. It's mostly just a best practice to avoid a hard-code value. @scottsand-db we can add a comment for the config key to warn the user.